packages/fastify/src/graphql.ts

```diff
@@ -48,40 +62,54 @@ export async function redwoodFastifyGraphQLServer(
         options.extraPlugins || []
       originalExtraPlugins.push(useRedwoodRealtime(options.realtime))
       options.extraPlugins = originalExtraPlugins
+
+      // uses for SSE single connection mode with the `/graphql/stream` endpoint
+      if (options.realtime.subscriptions) {
+        method.push('PUT')
+      }
     }
 
     const { yoga } = createGraphQLYoga(options)
 
-    // TODO: This should be refactored to only be defined once and it might not live here
-    // Ensure that each request has a unique global context
-    fastify.addHook('onRequest', (_req, _reply, done) => {
-      getAsyncStoreInstance().run(new Map<string, GlobalContext>(), done)
-    })
+    const graphQLYogaHandler = async (
+      req: FastifyRequest,
+      reply: FastifyReply
+    ) => {
+      const response = await yoga.handleNodeRequest(req, {
+        req,
+        reply,
+        event: transformToRedwoodGraphQLContextEvent(req),
+        requestContext: {},
+      })
+
+      for (const [name, value] of response.headers) {
+        reply.header(name, value)
+      }
+
+      reply.status(response.status)
+      reply.send(response.body)
+
+      return reply
+    }
+
+    const routePaths = ['', '/health', '/readiness', '/stream']
```

Comment:
Currently, I don't have HTTP1 SSE single connection mode stream working yet, but this is anticipation of the endpoint being needed. See: https://the-guild.dev/graphql/sse/get-started#with-fastify

```
import Fastify from 'fastify'; // yarn add fastify
import { createHandler } from 'graphql-sse/lib/use/fastify';
 
// Create the GraphQL over SSE handler
const handler = createHandler({ schema });
 
// Create a fastify app
const fastify = Fastify();
 
// Serve all methods on `/graphql/stream`
fastify.all('/graphql/stream', handler);
 
fastify.listen({ port: 4000 });
console.log('Listening to port 4000');
```

And it needs `PUT` as a http method.